components/html-block/html-block.js

```diff
@@ -177,15 +177,11 @@ class HtmlBlock extends LitElement {
 
 		if (!this._contentObserver || this.noDeferredRendering) return;
 
-		const template = this._findSlottedElement('TEMPLATE');
-		if (template) this._contentObserver.observe(template.content, { attributes: true, childList: true, subtree: true });
-		else {
-			const slot = this.shadowRoot.querySelector('slot');
-			if (slot) {
-				slot.assignedNodes({ flatten: true }).forEach(
-					node => this._contentObserver.observe(node, { attributes: true, childList: true, subtree: true })
-				);
-			}
+		const slot = this.shadowRoot.querySelector('slot');
+		if (slot) {
+			slot.assignedNodes({ flatten: true }).forEach(
```

Comment:
So situationally this can lead to `slot.assignedNodes` being called twice on render, specifically if an html-block gets re-added to the DOM, which will cause this to fire and then also fire the slotchange event. I don't know that this is avoidable though, since we can't exactly cache this (the nodes can certainly change before the slotchange event fires).